tests/integration/buildcmd/build_integ_base.py

```diff
@@ -13,7 +13,7 @@
 from pathlib import Path
 
 from samcli.yamlhelper import yaml_parse
-from tests.testing_utils import IS_WINDOWS
+from tests.testing_utils import IS_WINDOWS, _run_command
```

Comment:
Can we make `_run_command` not private since it is being imported into another file?

Reply:
Updated, this made PR little long, but it is mostly rename by by IDE.